File: app/java/src/main/java/com/example/production/domain/model/process/WorkOrderDetail.java
package com.example.production.domain.model.process;

import lombok.*;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WorkOrderDetail {
    private Integer id;
    private String workOrderNumber;
    private Integer sequence;
    private String processCode;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;

    // リレーション
    private WorkOrder workOrder;
    private Process process;
}

File: app/csharp/src/ProductionManagement.Domain/Models/Process/WorkOrderDetail.cs
namespace ProductionManagement.Domain.Models.Process;

/// <summary>
/// 作業指示明細データ
/// </summary>
public class WorkOrderDetail
{
    public int Id { get; set; }
    public required string WorkOrderNumber { get; set; }
    public required int Sequence { get; set; }
    public required string ProcessCode { get; set; }
    public DateTime? CreatedAt { get; set; }
    public DateTime? UpdatedAt { get; set; }
}
